installer/Migration/V5_2_0/LangStringHelper.php

```diff
@@ -24,6 +24,7 @@
 use OrangeHRM\Installer\Util\V1\Dto\LangString;
 use OrangeHRM\Installer\Util\V1\LanguageHelper;
 
+//TODO:: need to refactor this class
```

Comment:
@RajithaKumara I'll handle this in my next PR. Just need to have a common class neh?